collect_app/src/androidTest/java/org/odk/collect/android/storage/StorageMigrationCompletedBannerTest.java

```diff
@@ -21,18 +23,10 @@
 
 import static org.odk.collect.android.support.CollectHelpers.overrideAppDependencyModule;
 
+@RunWith(AndroidJUnit4.class)
```

Comment:
Why is this needed? Isn't that the default runner?

Reply:
You technically don't need it but I think it's weird not to have it. It is possible to specify other ones and the Google team loves deprecating/changing them.